src/spacel/provision/alarm/trigger/factory.py

```diff
@@ -85,7 +85,7 @@ def add_triggers(self, template, triggers, endpoint_resources):
             if dimensions:
                 alarm_properties['Dimensions'] = [
                     {'Name': k, 'Value': v} for k, v in dimensions.items()
-                ]
+                    ]
```

Comment:
I'm guessing my IDE did it, and I just had too big of a diff to notice+revert.

Refactored to something IDE and pylint can agree upon.